bot/cogs/snakes.py

```diff
@@ -15,31 +38,67 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
+    @staticmethod
+    def snake_url(name) -> str:
+        """Get the URL of a snake"""
+
+        def encode_url(text):
```

Comment:
The standard lib already provides a tool for this:
[urllib.parse.quote_plus](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus)